### PR TITLE
docs(custom-views): Add custom views documentation

### PR DESCRIPTION
**Pull request by MichaelSun48**
This PR adds documentation for the ~custom~ issue views feature. Also updates some saved searches documentation to signal that it is deprecated and slated for removal.

### CODE REVIEW

File: docs/product/issues/custom-views/index.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+#### Renaming views
+
+To rename a view, first select it from the tab list. Then, either double click on the view's name or click the ellipsis menu and select **Rename**, then type in a new name and hit enter.
+
+
+## Sharing views
+
+Views were built specifically with sharability in mind. To share a view with someone else within your organization, simply copy the link to your issue view and send it to them. Anyone that clicks on the link will be navigate to a **temporary tab** that contains the same query and sort as your view. They can then save the view as their own by clicking **Save View**, or discard it by clicking **Discard**
+
```
Comment (lizokm) [inline docs/product/issues/custom-views/index.mdx:53]:
```suggestion
We built views in a way that makes them easy to share with people in your organization. To share, just copy the link to your issue view and send it to a fellow org member. The link will take them to a **temporary tab** that contains the same query and sort as your view. They'll then be able to save the view as their own by clicking **Save View**, or discard it by clicking **Discard**.

```

**Review by lizokm — COMMENTED** [review]
Great first pass. You're a good writer and should not discount your skills! Let me know when you'd like me to take another look :)

File: docs/product/issues/custom-views/index.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+### Renaming Views
+
+To rename a view, select it from the tab list. Then, either double click on the view's name or click the ellipsis menu and select **Rename**, type in a new name, and hit enter.
+
+
+## Sharing Views
+
+We built views in a way that makes them easy to share with people in your organization. To share, just copy the link to your issue view and send it to a fellow org member. The link will take them to a **temporary tab** that contains the same query and sort as your view. They'll then be able to save the view as their own by clicking **Save View**, or discard it by clicking **Discard**.
+
```
Comment (MichaelSun48) [inline docs/product/issues/custom-views/index.mdx:52]: Reminder to self: add a note that temporary tabs are not persisted, unlike normal tabs.

File: docs/concepts/search/saved-searches.mdx [hunk 3]
```diff
@@ -5,6 +5,11 @@ keywords: ["saved search", "premade searches"]
 description: "Learn more about default, recommended, and saved searches."
 ---
 
+{/* Maybe this note should be yellow/red since there is some time pressure?  */}
+<Note>
+   Saved Searches is deprecated and will be removed at the end of the year. Your saved searches will still be available in the "Add View" menu on the **Issues** page, and they can easily be converted to a Issue Views.
+</Note>
+
```
Comment (lizokm) [inline docs/concepts/search/saved-searches.mdx:12]:
```suggestion
<Alert>
Saved Searches has been deprecated and will be removed at the end of the year. Your saved searches will still be available in the "Add View" menu on the **Issues** page. Any saved search can be converted to an Issue View.
</Alert>

```

File: docs/product/issues/issue-priority/index.mdx [hunk 4]
```diff
@@ -36,9 +36,9 @@ You can update issue priority manually at any time, either from the issue detail
 
 ## Priority in the Issue Stream
 
-By default, the issue stream on the **Issues** page filters out low-priority issues. If you'd like to be able to see low-priority issues, you can [set your default search](/product/reference/search/saved-searches/#set-a-default-search) to include them.
+The default view on the **Issues** page is the "Prioritized" view, which shows high and medium priority issues. But all views can be filtered to include, for example, only high priority issues by adding the query `issue.priority:high` to the search bar and saving the view.
```
Comment (lizokm) [inline docs/product/issues/issue-priority/index.mdx:39]:
```suggestion
The default view on the **Issues** page is the "Prioritized" view, which shows high and medium priority issues. But all views can be filtered to include, for example, only high priority issues. You can do this by adding the query: `issue.priority:high` to the search bar and saving the view.
```

File: docs/product/issues/issue-views/index.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+
+1. **Name your view**: Give your view a name that describes the types of issues it will show.
+2. **Set your view's search query**: Choose the search query that will be applied by default when you select this view. Don't forget to hit enter to apply the query!
+3. **Set your view's sort**: Select how you want the issues to be sorted in this view (for example, by date, events, and so on).
+4. **Save your view**: Click on the ellipsis menu within the tab and select **Save Changes**.
+
+<Note>
+If you had a default custom saved search, it'll be converted to an issue view and will appear as your default view automatically, just like before. Any other saved searches you created will be available in the **Add View** menu until the end of the yea. Please save them as views if you'd like to keep them.
+</Note>
```
Comment (lizokm) [inline docs/product/issues/issue-views/index.mdx:29]:
```suggestion
<Note>
If you had a default custom saved search, it'll be converted to an issue view and will appear as your default view automatically, just like before. Any other saved searches you created will be available in the **Add View** menu until the end of the year. Please save them as views if you'd like to keep them.
</Note>
```

File: docs/product/issues/issue-views/index.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Customizing Your Views
+
+
+### Updating View Queries and Sorting
+
+Views are defined by their search query and sort order, and you can safely change the query or sort within your view without changing the persistant query or sort. If you have inputted a different query from the one your view was saved with, you'll see an **unsaved changes indicator** (shown below). To update your view with this new query or sort, click the ellipsis menu and select **Save Changes**. To discard them, click **Discard Changes** within the ellipsis menu, or close your Sentry tab without saving.
+
```
Comment (lizokm) [inline docs/product/issues/issue-views/index.mdx:37]:
```suggestion
Views are defined by their search query and sort order, but you can safely change the query or sort within your view without changing the persistent query or sort. If you want to input a different query from the one your view was saved with, you'll see an **unsaved changes indicator** (shown below). To update your view with this new query or sort, click the ellipsis menu and select **Save Changes**. To discard them, click **Discard Changes** within the ellipsis menu, or close your Sentry tab without saving.

```

File: docs/product/issues/issue-views/index.mdx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### Renaming Views
+
+To rename a view, select it from the tab list. Then, either double click on the view's name or click the ellipsis menu and select **Rename**, type in a new name, and hit enter.
+
+
+## Sharing Views
+
+We built Issue Views in a way that makes them easy to share with people in your organization. To share, just copy the link to your issue view and send it to a fellow org member. The link will take them to a **temporary tab** that contains the same query and sort as your view. They'll then be able to save the view as their own by clicking **Save View**, or discard it by clicking **Discard**. Note that temporary tabs will not be persisted across page navigations or reloads.
```
Comment (lizokm) [inline docs/product/issues/issue-views/index.mdx:52]:
```suggestion
We built Issue Views in a way that makes them easy to share with people in your organization. To share, just copy the link to your issue view and send it to a fellow org member. The link will take them to a **temporary tab** that contains the same query and sort as your view. They'll then be able to save the view as their own by clicking **Save View**, or discard it by clicking **Discard**. Note, that temporary tabs will not be persisted across page navigations or reloads.
```

**Review by lizokm — APPROVED** [review]
Looks great! Just a few more minor tweaks.

File: docs/product/issues/issue-priority/index.mdx [hunk 8]
```diff
@@ -36,9 +36,9 @@ You can update issue priority manually at any time, either from the issue detail
 
 ## Priority in the Issue Stream
 
-By default, the issue stream on the **Issues** page filters out low-priority issues. If you'd like to be able to see low-priority issues, you can [set your default search](/product/reference/search/saved-searches/#set-a-default-search) to include them.
+The default view on the **Issues** page is the "Prioritized" view, which shows high and medium priority issues. But all views can be filtered to include, for example, only high priority issues. You can do this by adding the query: `issue.priority:high` to the search bar and saving the view.
```
Comment (leeandher) [inline docs/product/issues/issue-priority/index.mdx:39]: I'd try to retain indicating here that low prio can still be searched for, which feels like what the original text was going for. Maybe changing your example to `issue.priority:low`instead.

File: docs/product/issues/issue-views/img/issue_views_add_view.png
Comment (leeandher) [inline docs/product/issues/issue-views/img/issue_views_add_view.png:1]: nit: The call out in the screenshot says `Early Adopter only` which will become outdated when this releases. Might have to do some manual DOM editing to get around it, but will prevent the screenshot from becoming dated

File: docs/product/issues/issue-views/index.mdx
Comment (leeandher) [inline docs/product/issues/issue-views/index.mdx:1]: great work on this!

**Pull request merged**